keras2onnx/_builtin.py

```diff
@@ -1875,8 +1889,26 @@ def convert_tf_strided_slice(scope, operator, container):
     else:
         new_axis_unsqueeze = operator.inputs[0].full_name
 
+    data_shape = oopb.add_node('Shape',
+                                operator.inputs[0].full_name,
+                                operator.inputs[0].full_name + '_shape')
```

Comment:
always need a op_version for add_node method.

Reply:
done